ci/copy-windows-release-artifacts.sh

```diff
@@ -5,34 +5,38 @@ set -eou pipefail
 
 RELEASE_TAG=$1
 OUTPUT_DIR=$2
+SPLIT_RELEASE=${3:-false}
 
 mkdir -p $OUTPUT_DIR/github
 mkdir -p $OUTPUT_DIR/artifactory
 mkdir -p $OUTPUT_DIR/split-release
-INSTALLER="$OUTPUT_DIR/github/daml-sdk-$RELEASE_TAG-windows.exe"
-EE_INSTALLER="$OUTPUT_DIR/artifactory/daml-sdk-$RELEASE_TAG-windows-ee.exe"
-mv "bazel-bin/release/windows-installer/daml-sdk-installer-ce.exe" "$INSTALLER"
-mv "bazel-bin/release/windows-installer/daml-sdk-installer-ee.exe" "$EE_INSTALLER"
-chmod +wx "$INSTALLER"
-chmod +wx "$EE_INSTALLER"
 
-if ! [ -f /C/Users/u/.dotnet/tools/azuresigntool.exe ]; then
-    "/C/Program Files/dotnet/dotnet.exe" tool install --global AzureSignTool --version 3.0.0
-fi
+if [ "$SPLIT_RELEASE" = "false" ]; then
```

Comment:
fwiw given that this script is version specific and pushing it doesn’t do any harm we could also keep it in here until we drop it completely instead of complicating the logic. But happy either way.

Reply:
IIUC the split process still relies on the last 6 lines of this script, but, yes, this PR is pretty much a marker for future deletion.

Reply:
Sorry for being unclear, I meant drop the windows installer completely not the whole script.

Reply:
Right, we're in alignment then.